src/components/Canvas/Editor.tsx

```diff
@@ -1158,25 +1225,53 @@ export default class Editor extends EventDispatcher {
   };
 
   // this will be only used by the current device user
-  private drawPixelInInteractionLayer(rowIndex: number, columnIndex: number) {
+  private drawPixelInInteractionLayer(
+    rowIndex: number,
+    columnIndex: number,
+    brushPattern: Array<Array<1 | 0>>,
```

Comment:
Is it correct that `1` represents the pixel to be painted and `0` represents the pixel to be ignored? It was good to understand because the code was intuitive and well written, but it would be nice to reveal your intentions with comments or const.

Reply:
You are right on track. 1 represents the pixel to be painted, and 0 means it will be blank. I will change this to enum!